string name
int16 duration
int16 head_lr
int16 head_ud
bool head_init
---
uint16[] data